src/status_im/transport/inbox.cljs

```diff
@@ -305,34 +328,24 @@
                 {:db (assoc-in db [:transport.inbox/topics topic] inbox-topic)
                  :data-store/tx [(transport-store/save-transport-inbox-topic-tx
                                   {:topic topic
-                                   :inbox-topic inbox-topic})]}
-                (request-messages topic)))))
+                                   :inbox-topic inbox-topic})]}))))
 
 (fx/defn resend-request
   [{:keys [db] :as cofx} {:keys [request-id]}]
-  (let [{:keys [from to topic]} (get-in db [:transport.inbox/requests request-id])]
-    (log/info "offline inbox: message request" request-id " expired for inbox topic"  topic "from" from "to" to)
-    (fx/merge cofx
-              {:db (-> db
-                       (update :transport.inbox/requests dissoc request-id)
-                       (update-in [:transport.inbox/topics topic] dissoc :request-pending?))}
-              (request-messages topic))))
-
-(fx/defn add-request
-  [{:keys [db] :as cofx} {:keys [topic request-id from to]}]
-  (log/info "offline inbox: message request " request-id "sent for inbox topic" topic "from" from "to" to)
-  {:db (-> db
-           (assoc-in [:transport.inbox/requests request-id] {:from from
-                                                             :to to
-                                                             :topic topic})
-           (assoc-in [:transport.inbox/topics topic :request-pending?] true))})
+  (if (= maximum-number-of-attemps
```

Comment:
that shouldn't be a problem, but can we make it `<=` for future generations :D

Reply:
why ?

Reply:
idk, someone will add accidentally add another `inc` for this value and condition will never catch "too many attempts"